docs/administration/deployment/deploying-locally/index.md

```diff
@@ -17,9 +17,9 @@ $ firefox https://helm.sh/docs/intro/install/
 **2.** Deploy REANA cluster:
 
 ```console
-$ wget https://raw.githubusercontent.com/reanahub/reana/maint-0.7/etc/kind-localhost-30443.yaml
+$ wget https://raw.githubusercontent.com/reanahub/reana/0.8.0/etc/kind-localhost-30443.yaml
 $ kind create cluster --config kind-localhost-30443.yaml
-$ wget https://raw.githubusercontent.com/reanahub/reana/maint-0.7/scripts/prefetch-images.sh
+$ wget https://raw.githubusercontent.com/reanahub/reana/0.8.0/scripts/prefetch-images.sh
```

Comment:
It is better to use `maint-0.8` rather than the fixed tag `0.8.0`, because we might have some bug fixes coming in the next days... In such a case, the instructions wouldn't work when we shall be in-between 0.8.0 and 0.8.1 point releases. Whilst having `maint-0.8` is safer, since it is moving target, and the committed bug fixes work "immediately" there, regardless of point releases. 

(That's basically why we went for `maint-x.y` there.)

Reply:
Updated and branch `maint-0.8` pushed to `reana`.